packages/core/src/index.ts

```diff
@@ -39,11 +81,15 @@ const eventSchema = {
   },
 };
 
-server.post<{ Body: EventReceivedType; Reply: any }>("/reset", { schema: eventSchema }, async (request, reply) => {
-  const { event } = request.body;
-  store.set(event, null);
-  reply.status(200).send({ status: "ok" });
-});
+server.post<{ Body: EventReceivedType; Reply: any }>(
+  "/reset",
```

Comment:
Please move it to backdoor or /admin route or whatever to make it a bit separated. Makes sense? Alternative design would be to force our clients to use `/mocks/{your-mocked-route}` route, then we can have everything at top level. IMO this route should be customisable in the future.

/mocks/

Reply:
Moved